harness/determined/common/schemas/expconf/_gen.py

```diff
@@ -1480,7 +1480,7 @@
                 "array",
                 "null"
             ],
-            "default": [],
+            "default": null,
```

Comment:
Looking at https://pkg.go.dev/encoding/json, this is probably right.   The default value for a struct on omitempty would be a nil, not an empty array.

If the slurm section was present then the sbatch_args default value would be [] though.

> The "omitempty" option specifies that the field should be omitted from the encoding if the field has an empty value, defined as false, 0, a nil pointer, a nil interface value, and any empty array, slice, map, or string.